app/serializers/organization_serializer.rb

```diff
@@ -5,10 +5,11 @@ class OrganizationSerializer
   include MediaLinksSerializer
   include CachedSerializer
 
-  attributes :id, :display_name, :description, :introduction, :title, :href, :primary_language, :listed_at
+  attributes :id, :display_name, :description, :introduction, :title, :href, :primary_language, :listed_at, :slug
   optional :avatar_src
   media_include :avatar, :background
   can_include :organization_contents, :organization_roles, :projects, :owners
+  can_filter_by :display_name, :slug, :listed_at
```

Comment:
Check the query path if there is a more restrictive where clause this may not be needed. If ti's a straight match on this then without an index it'll probably be a table scan.